data/constants/keycodes/keycodes_0.0.1.hjson

```diff
@@ -36,7 +36,9 @@
         "0x52C0/0x001F": {
             "define": "QK_LAYER_TAP_TOGGLE"
         },
-        // 0x52E0/0x001F - UNUSED
+        "0x52E0/0x001F": {
+            "define": "QK_PERSISTENT_DEF_LAYER"
+        },
```

Comment:
This range by itself would need to go into `keycodes_0.0.4.json`, 0.0.1 is already locked in and immutable.
Then `./util/regen.sh` to update everything else.

Reply:
Thank you for the comment. I have moved the keycode definition to `keycodes_0.0.4.json` in commit 757a27bbb6fe43edf80f4bbdfbb8b67ef0bc4f03. Running `util/regen.sh` afterwards did not change anything, which is good, since it means that the first regeneration worked well.